core/core_integrations/messenger.js

```diff
@@ -0,0 +1,198 @@
+var fs = require('fs'),
+	path = require('path'),
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 space characters but found 0.](https://www.codacy.com/app/theprojectshop/Kassy/file/2575682484/issues/source?bid=3282987&fileBranchId=3288129#l2)